packages/schemas/alterations/next-1751337183-add-require-mfa-on-sign-in-to-users.ts

```diff
@@ -0,0 +1,20 @@
+import { sql } from '@silverhand/slonik';
+
+import type { AlterationScript } from '../lib/types/alteration.js';
+
+const alteration: AlterationScript = {
+  up: async (pool) => {
+    await pool.query(sql`
+      alter table users
+      add column require_mfa_on_sign_in boolean not null default true;
```

Comment:
Don't forget to move this column under `logto_config` before we launch this to production.  Otherwise, it will be breaking change.